src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -30,51 +34,61 @@ if not exist "%DOTNET_CMD%" (
+)
+
 @echo on
-call "%DOTNET_CMD%" restore "%TOOLRUNTIME_PROJECTJSON%" %TOOLRUNTIME_RESTORE_ARGS%
+call "%DOTNET_CMD%" restore "%TOOLRUNTIME_PROJECT%" %TOOLRUNTIME_RESTORE_ARGS%
 set RESTORE_ERROR_LEVEL=%ERRORLEVEL%
 @echo off
 if not [%RESTORE_ERROR_LEVEL%]==[0] (
-	echo ERROR: An error occured when running: '"%DOTNET_CMD%" restore "%TOOLRUNTIME_PROJECTJSON%"'. Please check above for more details.
-	exit /b %RESTORE_ERROR_LEVEL%
+  echo ERROR: An error occured when running: '"%DOTNET_CMD%" restore "%TOOLRUNTIME_PROJECT%"'. Please check above for more details.
+  exit /b %RESTORE_ERROR_LEVEL%
 )
 @echo on
-call "%DOTNET_CMD%" publish "%TOOLRUNTIME_PROJECTJSON%" -f netcoreapp1.0 -r %BUILDTOOLS_TARGET_RUNTIME% -o "%TOOLRUNTIME_DIR%"
+call "%DOTNET_CMD%" publish "%TOOLRUNTIME_PROJECT%" -f %PUBLISH_TFM% -r %BUILDTOOLS_TARGET_RUNTIME% -o "%TOOLRUNTIME_DIR%"
 set TOOLRUNTIME_PUBLISH_ERROR_LEVEL=%ERRORLEVEL%
 @echo off
 if not [%TOOLRUNTIME_PUBLISH_ERROR_LEVEL%]==[0] (
-	echo ERROR: An error ocurred when running: '"%DOTNET_CMD%" publish "%TOOLRUNTIME_PROJECTJSON%" -f netcoreapp1.0'. Please check above for more details.
-	exit /b %TOOLRUNTIME_PUBLISH_ERROR_LEVEL%
+  echo ERROR: An error ocurred when running: '"%DOTNET_CMD%" publish "%TOOLRUNTIME_PROJECT%" -f %PUBLISH_TFM%'. Please check above for more details.
+  exit /b %TOOLRUNTIME_PUBLISH_ERROR_LEVEL%
 )
 @echo on
-call "%DOTNET_CMD%" publish "%TOOLRUNTIME_PROJECTJSON%" -f net46 -r %BUILDTOOLS_NET46_TARGET_RUNTIME% -o "%TOOLRUNTIME_DIR%\net46"
+call "%DOTNET_CMD%" publish "%TOOLRUNTIME_PROJECT%" -f net46 -r %BUILDTOOLS_NET46_TARGET_RUNTIME% -o "%TOOLRUNTIME_DIR%\net46"
 set NET46_PUBLISH_ERROR_LEVEL=%ERRORLEVEL%
 @echo off
 if not [%NET46_PUBLISH_ERROR_LEVEL%]==[0] (
-	echo ERROR: An error ocurred when running: '"%DOTNET_CMD%" publish "%TOOLRUNTIME_PROJECTJSON%" -f net46'. Please check above for more details.
-	exit /b %NET46_PUBLISH_ERROR_LEVEL%
+  echo ERROR: An error ocurred when running: '"%DOTNET_CMD%" publish "%TOOLRUNTIME_PROJECT%" -f net46'. Please check above for more details.
+  exit /b %NET46_PUBLISH_ERROR_LEVEL%
 )
 
 :: Microsoft.Build.Runtime dependency is causing the MSBuild.runtimeconfig.json buildtools copy to be overwritten - re-copy the buildtools version.
 Robocopy "%BUILDTOOLS_PACKAGE_DIR%\." "%TOOLRUNTIME_DIR%\." "MSBuild.runtimeconfig.json"
 
 :: Copy Portable Targets Over to ToolRuntime
 if not exist "%PACKAGES_DIR%\generated" mkdir "%PACKAGES_DIR%\generated"
-set PORTABLETARGETS_PROJECTJSON=%PACKAGES_DIR%\generated\project.json
-echo %MSBUILD_CONTENT_JSON% > "%PORTABLETARGETS_PROJECTJSON%"
+set PORTABLETARGETS_PROJECT=%PACKAGES_DIR%\generated\project.json
+echo %MSBUILD_PROJECT_CONTENT% > "%PORTABLETARGETS_PROJECT%"
 @echo on
-call "%DOTNET_CMD%" restore "%PORTABLETARGETS_PROJECTJSON%" %INIT_TOOLS_RESTORE_ARGS% --packages "%PACKAGES_DIR%\."
+call "%DOTNET_CMD%" restore "%PORTABLETARGETS_PROJECT%" %INIT_TOOLS_RESTORE_ARGS% --packages "%PACKAGES_DIR%\."
```

Comment:
It's a bit confusing because the `MSBUILD_PROJECT_CONTENT` (formerly `MSBUILD_CONTENT_JSON`) actually gets put into the `PORTABLETARGETS_PROJECT` file... There's just one generated project.json/csproj here. I'll at least rename the "content" variable so it matches the file name.